bin/oneoff/data_fix/reset_race_data

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+#
+# Script to reset the race data for the subset of users affected by the race
+# interstitial bug
+#
+
+require_relative '../../../dashboard/config/environment'
+
+File.readlines(ARGV[0]).each do |line|
+  id = line.to_i
+  user = User.find_by_id(id)
+  if user.nil?
+    puts("user #{id} does not exist")
+    next
+  end
+
+  unless user.races
+    puts("user #{id} has no race data")
+    next
+  end
+
+  user.races = nil
+  user.save!
```

Comment:
In addition to rescuing/logging errors, maybe log the id of every User that is successfully saved?  That helps track what went right/wrong after the script has been run.